homeassistant/components/sensor/yweather.py

```diff
@@ -139,6 +139,9 @@ def device_state_attributes(self):
     def update(self):
         """Get the latest data from Yahoo! and updates the states."""
         self._data.update()
+        if len(self._data.yahoo.RawData) == 0:
```

Comment:
Actually, if `RawData` is a dictionary, empty dictionaries are considered false, so no need for `len`.

```python
> bool({})
False
> bool({'hello': 'world'})
True
```